presto-spi/src/main/java/com/facebook/presto/spi/type/Type.java

```diff
@@ -78,7 +78,7 @@
      * {@code position}. This is the value returned to the user via the
      * REST endpoint and therefore must be JSON serializable.
      */
-    Object getObjectValue(ConnectorSession session, Block block, int position);
+    Object getJsonSerializableObjectValue(ConnectorSession session, Block block, int position);
```

Comment:
I think part of the reason this was being used incorrectly in tests was that the name was not descriptive enough.